cli/crawl-cli.js

```diff
@@ -52,7 +52,7 @@ async function run(inputUrls, outputPath, verbose, logPath, numberOfCrawlers, da
     const startTime = new Date();
 
     reporters.forEach(reporter => {
-        reporter.init({verbose, startTime, urls: inputUrls.length, logPath});
+        reporter.init({verbose, startTime, urls: inputUrls.length, logPath, proxyHost});
```

Comment:
do you really need proxyHost? Why not regionCode?

Reply:
This was just one way of putting something useful in this column without having to try to expose a new CLI param or environment variable. Ideally one would provide a region code via CLI, yes.

Reply:
but we do provide region code via CLI:

> -r, --region-code <region>